src/tests/ftest/io/ior_small.py

```diff
@@ -48,4 +48,32 @@ def test_ior_small(self):
 
         :avocado: tags=all,daosio,small,pr,hw,iorsmall
         """
+        flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
+        apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        transfer_size = self.params.get("ior_transfer_size",
+                                        '/run/ior/iorflags/*')
+        block_size = self.params.get("ior_block_size", '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+
+        for oclass in obj_class:
+            self.ior_cmd.daos_oclass.update(oclass)
+            for api in apis:
+                self.ior_cmd.api.update(api)
+                self.ior_cmd.flags.update(flags[0])
+                for ts in transfer_size:
+                    self.ior_cmd.transfer_size.update(ts)
+                    if ts == '256B':
```

Comment:
There is a mapping between transfer_size and block_size. To avoid the if condition, wouldn't it be easier to read comma separated transfer_size,blocksize in the yaml and split it here ? That way we wouldn't need if condition and more variants can be added in future without having to add additional elsif conditions.